### ISSUE TITLE
va-file-input: Delete this file modal yes/no buttons are not reachable on iOS/VoiceOver/Safari

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/vets-design-system-documentation#4722]
# Bug Report
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened
You can activate the X close button and close out the modal.
You can't navigate to the yes/no buttons. If you try to move forward to the yes/no buttons focus jumps out of the modal to somewhere else. On Storybook it jumps down to the controls. On the Claim Status tool it jumps out of the modal.

## What I expected to happen
You should be able to navigate to the “yes, remove this” & “no, keep this” buttons and activate them.

## Reproducing

- CSS Library version: Current
- Device: (e.g. iPhone 8, Macbook) iOS/VoiceOver/Safari, haven't tried Android/TalkBack
- Browser: (e.g. Firefox, IE 11)

Steps to reproduce:

1. On an iphone open https://design.va.gov/storybook/?path=/story/uswds-va-file-input--default in Safari.
2. Upload a file
3. Turn on VoiceOver
4. Navigate to the Delete button. Activate it to open the modal
5. Try to navigate to the yes and no buttons

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

**Comment by RyanMunsch** [comment]
This will remain open into next sprint due to pending PR reviews.
